apps/ledger-live-desktop/tests/models/AccountPage.ts

```diff
@@ -27,6 +32,17 @@ export class AccountPage {
     this.stakeButtonCosmos = page.locator("data-test-id=stake-button-cosmos");
     this.stakeBanner = page.locator("data-test-id=account-stake-banner");
     this.stakeBannerButton = page.locator("data-test-id=account-stake-banner-button");
+    this.receiveButton = page
+      .locator("data-test-id=account-buttons-group")
```

Comment:
why do we have here both locator with test-id and identification of element by role? wouldn't only one of those sufficient?

Reply:
The reason for using two locators is to ensure that the correct "Receive" button is selected, especially in cases where there are multiple "Receive" buttons on the page (such as ETH). The first part of the locator, `page.locator("data-test-id=account-buttons-group")`, narrows down the scope to a specific group of buttons within the account section. The second part, `.getByRole("button", { name: "Receive" })`, selects the "Receive" button within that group. This approach avoids ambiguity and ensures that the correct button is interacted with.

Reply:
why not locating it by text in this case?